packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader.component.tsx

```diff
@@ -116,11 +96,55 @@ const CameraMediaUploaderModal: React.FC<CameraMediaUploaderModalProps> = ({
         handleTakePhoto,
         error,
         setError,
+        allowedExtensions,
       }}
     >
       {returnComponent}
     </CameraMediaUploaderContext.Provider>
   );
 };
 
+const CameraMediaUploadTabs = () => {
+  const { t } = useTranslation();
+  const [view, setView] = useState('upload');
+  const { cameraOnly } = useContext(CameraMediaUploaderContext);
+  const mediaStream = useRef<MediaStream | undefined>();
+
+  const stopCameraStream = useCallback(() => {
+    mediaStream.current?.getTracks().forEach((t) => t.stop());
+  }, [mediaStream]);
+
+  useEffect(() => {
+    return () => {
+      stopCameraStream();
+    };
+  }, [stopCameraStream]);
+
+  if (cameraOnly) {
+    return <CameraComponent mediaStream={mediaStream} stopCameraStream={stopCameraStream} />;
+  }
+
+  return (
+    <div className={styles.cameraSection}>
```

Comment:
I'm curious why this is missing the <ComposedModal /> wrapping component?

Reply:
The component `CameraMediaUploadTabs` wrapping this render is called by CameraMediaUploader on line 79 same file, which is then rendered on line 102.
Subsequently it is wrapped by the Context.